cfgmgr/macsecmgrd.cpp

```diff
@@ -45,6 +46,20 @@ string gResponsePublisherRecordFile;
 /* Global database mutex */
 mutex gDbMutex;
 
+bool received_sigterm = false;
+static struct sigaction old_sigaction;
+
+void sig_handler(int signo)
+{
+    SWSS_LOG_ENTER();
+
+    if (old_sigaction.sa_handler != SIG_IGN && old_sigaction.sa_handler != SIG_DFL) {
+        old_sigaction.sa_handler(signo);
```

Comment:
Why we need to save `old_sigaction` and call its handler?

Reply:
The old signal action may be registered by other modules.
Just to keep the logic consistency with other modules. Please refer to: teamsyncd.cpp teammgrd.cpp natmgrd.cpp.

Reply:
IMO, signal handler should be simple and straightforward, calling old signal action in new signal action brings uncertainty.